src/components/Button/Button.jsx

```diff
@@ -1,11 +1,23 @@
 import "./Button.css";
 
-export const Button = ({buttonType, isDisabled, onClickFunction, children}) => {
+export const Button = ({
+  buttonType,
+  isDisabled,
+  onClickFunction,
+  children,
+}) => {
   const handleClick = () => {
     onClickFunction();
   };
 
   return (
-    <button className="button" buttonType={buttonType} disabled={isDisabled} onClick={handleClick}>{children}</button>
+    <button
+      className="button"
+      buttonType={buttonType}
+      disabled={isDisabled}
+      onClick={handleClick}
+    >
+      {children}
+    </button>
   );
-}
+};
```

Comment:
We don't need to commit this file because we already have a `Button` component. You should delete it.